exporter/exporterhelper/metricshelper.go

```diff
@@ -65,6 +65,9 @@ func NewMetricsExporter(exporterName string, pushMetricsData PushMetricsData, op
 	}
 
 	opts := newExporterOptions(options...)
+	if opts.recordMetrics {
```

Comment:
Nit: https://github.com/open-telemetry/opentelemetry-service/pull/315/files#diff-2cb0cce225c9d4239073b73773aadab9R56 can be removed.

Reply:
???

Reply:
https://github.com/open-telemetry/opentelemetry-service/pull/315/files#diff-2cb0cce225c9d4239073b73773aadab9L56
`// TODO: Add support for recordMetrics.`
This todo can be removed.

Reply:
Done.